backend/functions/mod_running_total.py

```diff
@@ -13,10 +13,11 @@
     protein -> float (amount of protein in grams in food item)
     sugar -> float (amount of sugar in the food item)
     username -> string (username of user)
+    adding -> bool (true if values are being added and false if subtracted)
 '''
 
 
-def addRunningTotal(calories, fat, carbs, protein, sugar, username):
+def modRunningTotal(calories, fat, carbs, protein, sugar, username, adding):
```

Comment:
Maybe you can name this modifyRunningTotal just to make it more clear what the function is doing. It can be confused with modulus.